lib/openstudio/workflow/jobs/run_preprocess.rb

```diff
@@ -37,7 +37,9 @@ def perform
     FileUtils.mkdir_p(@registry[:run_dir])
 
     # save the pre-preprocess file
-    File.open("#{@registry[:run_dir]}/pre-preprocess.idf", 'w') { |f| f << @registry[:model_idf].to_s }
+    if !@options[:fast]
```

Comment:
@shorowit this seems similar to the expandobjects job, can we add this in the osw too?

Reply:
skip_energyplus_preprocess

Reply:
@macumber Just to be clear, this is a workflow-gem step where it adds a bunch of OS objects to the model for reporting purposes. This is not an EnergyPlus workflow step like ExpandObjects. It doesn't seem to me like it's worth adding as a run option.

Reply:
It is adding output requests from reporting measures.  In my mind the same osw should run in both fast and normal modes.  But if this step is skipped then some osw's might not work in fast mode.  That was the test I was using in my head.

Reply:
I guess I was assuming that we would be the only one using fast mode, since it's a hidden argument. But if you really want me to do it, I can do it.